mdetsims/sim_utils.py

```diff
@@ -334,7 +352,7 @@ def _extra_init_for_wldeblend(self):
 
         LOGGER.info('catalog density: %f per sqr arcmin', self.ngal)
 
-    def get_mbobs(self, return_truth_cat=False):
+    def get_mbobs(self, return_truth_cat=True):
```

Comment:
Make sure to keep the default the same here.

Reply:
Forgot to change this back.